src/js/measurements.ts

```diff
@@ -48,10 +48,23 @@ const _patchStartTransaction = (
     transactionContext: TransactionContext,
     customSamplingContext?: CustomSamplingContext
   ): Transaction {
+    if (!transactionContext.op) {
+      transactionContext.op = 'default';
```

Comment:
M: Let's make a `const` for `default`, its being used in multiple laces.

Reply:
Let's add a comment explaining why we have to patch and set the `op` as `default`.
Same for `startChild`.